app/upgrades/v16/proposals.go

```diff
@@ -9,21 +9,15 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	govkeeper "github.com/cosmos/cosmos-sdk/x/gov/keeper"
 	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types/v1"
+	incentives "github.com/evmos/evmos/v16/x/incentives/types"
 )
 
-// deprecatedProposals is a map of the TypeURL
-// of the deprecated proposal types
-var deprecatedProposals = map[string]struct{}{
-	"/evmos.incentives.v1.RegisterIncentiveProposal": {},
-	"/evmos.incentives.v1.CancelIncentiveProposal":   {},
-	"/evmos.erc20.v1.RegisterCoinProposal":           {},
-}
-
-// DeleteDeprecatedProposals deletes the RegisterCoin, RegisterIncentives & CancelIncentiveProposal
-// proposals from the store because were deprecated
-func DeleteDeprecatedProposals(ctx sdk.Context, gk govkeeper.Keeper, logger log.Logger) {
+// DeleteIncentivesProposals deletes the RegisterIncentives & CancelIncentiveProposal proposals from the store
+// because the module was deprecated
+func DeleteIncentivesProposals(ctx sdk.Context, gk govkeeper.Keeper, logger log.Logger) {
```

Comment:
the latest change is the one that has the `DeleteDeprecatedProposals`, the current one on main branch. The `DeleteDeprecatedProposals` includes the incentives proposals

Reply:
ohhh okay I see, I didn't catch that - I thought this was still deleting the `RegisterCoinProposal` 👍 thanks for pointing that out 🙏

Reply:
ahhh no wait @GAtom22 - I didn't remember just when I commented that, but the reason why I intentionally applied these changes is that this is all in the `v16` handler. Why should he have logic in the v16 upgrade directory, when the v16 upgrade is already done and has different code? 🤔 

Should we rather port the `DeleteDeprecatedProposals` logic to the v17 handler?

Reply:
yeah, you're right. We'll have to add the logic to remove the RegisterCoinProposals on v17 upgrade handler